javascript/src/main/components/BasicCourseSearch/CourseFilters.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Form } from "react-bootstrap";
+
+const CourseFilters = ({ cancelled, handleCancelledOnChange, closed, handleClosedOnChange, full, handleFullOnChange }) => {
+
+    return (
+        <Form.Group controlId="BasicSearch.Hide">
+            <Form.Check type="checkbox" label="Cancelled" value={cancelled} onChange={handleCancelledOnChange} id="inline-checkbox-1"/>
+            <Form.Check type="checkbox" label="Closed" value={closed} onChange={handleClosedOnChange} id="inline-checkbox-2"/>
```

Comment:
I wonder whether it would be better to make these three id's meaningful, i.e. 

`id="inline-checkbox-cancelled"` instead of `id="inline-checkbox-1"`
`id="inline-checkbox-closed"` instead of `id="inline-checkbox-2"`
`id="inline-checkbox-full"` instead of `id="inline-checkbox-3"`

Reply:
Done!